docs/submitting-molecular-data.md

```diff
@@ -2,11 +2,86 @@
 id: submitting-molecular-data
 title: Submitting Molecular Data
 ---
+Molecular data will be submitted to your local Regional Data Processing Centre (RDPC). RDPCs are responsible for processing your programs molecular data according to the [Harmonization Pipeline](). If you are unsure what the correct RDPC is to submit to, please [contact the DCC](https://platform-ui.qa.argo.cancercollaboratory.org/contact).
 
-## Installing the Song and Score Clients
 
-## Formatting Molecular Metadata Payloads
+This guide will describe how to submit molecular data to the ARGO Data Platform. Molecular data consists of all raw data  files generated from your program's donors, as well as any associated file metadata.
```

Comment:
remove extra space between "data" and "files"

"Molecular data consists of all raw data  files generated from your program's donors"
is this specific enough? like should it say 
"Molecular data consists of all raw data files generated from your donor's sequencing processes"

or something like that.  Saying generated from your program's donors isn't clear.

Reply:
no bc images are also considered "raw files" and thats not about sequencing....

Reply:
How about this:
```
Molecular data consists of all raw data files generated from your program's donors (e.g. sequencing reads, slide images), as well as any associated file metadata.
``

Reply:
yes that's good!